src/offline_audio_context.rs

```diff
@@ -106,7 +106,10 @@ fn constructor(ctx: CallContext) -> Result<JsUndefined> {
 fn get_current_time(ctx: CallContext) -> Result<JsNumber> {
     let js_this = ctx.this_unchecked::<JsObject>();
     let napi_obj = ctx.env.unwrap::<NapiOfflineAudioContext>(&js_this)?;
-    let obj = napi_obj.unwrap();
+    let obj = match napi_obj.0.as_ref() {
+        Some(v) => v,
+        None => return ctx.env.create_double(0.),
+    };
```

Comment:
I was a bit surprised about this change at first, but I guess it prevents crashing when we do something like this?

```js
const result = await offlineContext.startRendering();
const now = offlineContext.currentTime
```

Seems that all this `Option` stuff was required to work around:

```rs
pub fn start_rendering_sync(self) -> AudioBuffer
```

Did you think it could be resolved at the rust level to have something like this instead:

```rs
pub fn start_rendering_sync(&self) -> AudioBuffer
```

Reply:
Yes definitely, let's change that API. And also implement https://webaudio.github.io/web-audio-api/#dom-offlineaudiocontext-suspend because it is used so much in the wpt.